tests/Differ/SebastianBergmannDifferTest.php

```diff
@@ -77,7 +77,7 @@ public function __toString()
 @@ @@
 -<?php namespace Foo\Bar;
 +<?php
- 
```

Comment:
You can write the code using concatenation of strings then, to avoid this, like @keradus suggested for me in my last PR.